apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -31,169 +32,100 @@ export default async function Checkout({ params }: { params: CheckoutParams }) {
-                'next-payment-legal-link-stripe-3',
+                'next-new-user-sign-in-link-2',
                 {
                   elems: {
-                    stripePrivacyLink: (
-                      <a href="https://stripe.com/privacy">
-                        Stripe privacy policy
-                      </a>
-                    ),
-                  },
-                },
-                <a href="https://stripe.com/privacy">Stripe privacy policy</a>
-              )}
-            </p>
-          </div>
-          <div>
-            <h4>Get Element - With reference</h4>
-            <p className="text-sm">
-              {l10n.getFragmentWithSource(
-                'next-payment-confirm-with-legal-links-static-3',
-                {
-                  elems: {
-                    termsOfServiceLink: (
-                      <a href="https://stripe.com/privacy">
-                        Stripe privacy policy
-                      </a>
-                    ),
-                    privacyNoticeLink: (
-                      <a href="https://stripe.com/privacy">
-                        Stripe privacy policy
-                      </a>
+                    a: (
+                      <button className="underline text-grey-400 hover:text-grey-400">
+                        Sign in
+                      </button>
                     ),
                   },
                 },
                 <>
-                  I authorize Mozilla to charge my payment method for the amount
-                  shown, according to{' '}
-                  <a href="https://www.example.com">Terms of Service</a> and{' '}
-                  <a href="https://www.example.com">Privacy Notice</a>, until I
-                  cancel my subscription.
+                  Already have a Mozilla account?&nbsp;
+                  <button className="underline text-grey-400 hover:text-grey-400">
+                    Sign in
+                  </button>
                 </>
               )}
             </p>
+          </form>
+
+          <hr className="mx-auto w-full border-grey-200" />
+
+          <div className="h-64 text-center flex items-center justify-center">
+            {'<placeholder>Passwordless signup</placeholder>'}
           </div>
-        </section>
-        <StripeWrapper
-          amount={fakeCart.amount}
-          currency={fakeCart.nextInvoice.currency}
-          cart={cart}
-        />
-      </section>
-    </>
+
+          <hr className="mx-auto w-full border-grey-200" />
+        </>
+      )}
+
+      {!session ? (
+        <h4
+          className="font-semibold text-grey-600 text-lg mt-14 mb-5"
+          data-testid="header-prefix"
+        >
+          {l10n.getString(
+            'payment-method-header-second-step-next',
+            '2. Choose your payment method2'
+          )}
+        </h4>
+      ) : (
+        <h4
+          className="font-semibold text-grey-600 text-lg mt-15 mb-5"
```

Comment:
The `<h4>` above has `mt-14` but this has `mt-15`.

Reply:
Ah yes, thank you!